index.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
Is this a recommended file name, index.php? If not, maybe a more plugin specific one would be nice

Reply:
mayby later when I have got an idea for a different name

Reply:
"In case of many users having roles in category context there probably should be a mechanism not showing all users at once if there are more than x users." In first step the maximum will be configurable by adminsettings